src/commands/math/advancedSymbols.js

```diff
@@ -3,7 +3,6 @@
  ***********************************/
 
 LatexCmds.notin =
-LatexCmds.sim =
```

Comment:
Does this mean that users of stock mathquill will no longer be able to enter this is \sim?